src/main/frontend/src/javascripts/pageEdit.js

```diff
@@ -23,14 +23,21 @@ $(function() {
             if (parentId !== null)
                 url = '/' + projectOwner + '/' + projectSlug + '/pages/' + parent.data('slug') + '/' + slugify(pageName) + '/edit';
         }
+        var csrfValue = $('input[type=hidden][name=_csrf]');
```

Comment:
do we really need to handle this manually? that sound ewww
ore didnt do it, so surely theres a better way

Reply:
can just remove all this after you rebased, I fixed it on master